### PR TITLE
lib/api, lib/model: Fixes around event request tracking

### PR DESCRIPTION
**Pull request by imsodin**
Two small things:

1. When refactoring the api I clearly didn't grok the function of the "minute-variable" in the folder summary service. Luckily I only changed the name but not semantics, so nothing broke but the name was completely off.

2. We don't register all calls to the event endpoint, e.g. not those to the `event/disk` endpoint. I don't see why we'd do that, thus I moved the `OnEventSub` call to the generic `getEvents` function.

### CODE REVIEW

File: lib/api/api.go [hunk 1]
```diff
@@ -1213,7 +1213,9 @@ func (s *service) getDiskEvents(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *service) getEvents(w http.ResponseWriter, r *http.Request, eventSub events.BufferedSubscription) {
-	s.fss.OnEventRequest()
+	if eventSub.Mask()&events.FolderSummary != 0 {
```
Comment (calmh) [inline lib/api/api.go:1216]: or `events.FolderCompletion`

File: lib/model/folder_summary.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 	"github.com/syncthing/syncthing/lib/util"
 )
 
-const minSummaryInterval = time.Minute
+const maxDurSinceLastEventReq = time.Minute
```
Comment (mikeee) [inline lib/model/folder_summary.go:25]: `maxDurationSinceLastEventReq` for the sake of verbosity seems reasonable perhaps?

File: lib/events/events.go [hunk 3]
```diff
@@ -238,6 +238,7 @@ type Event struct {
 type Subscription interface {
 	C() <-chan Event
 	Poll(timeout time.Duration) (Event, error)
+	Mask() EventType
```
Comment (calmh) [inline lib/events/events.go:241]: Maybe `Allows(EventType) bool` would be cleaner (just moving the `& ... != 0`)?

Comment (imsodin) [reply lib/events/events.go:241]: `Allows` feels kind of weird in this context, and e.g. `SubscribedTo` would be even worse. The mask value is also used as a map key in the api, i.e. there could be a use-case for actually getting the value. I think I'll choose simple over clean here.

**Pull request merged**